egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -35,7 +35,11 @@ def AddInputLayer(config_lines, feat_dim, splice_indexes=[0], ivector_dim=0):
     output_dim += len(splice_indexes) * feat_dim
     if ivector_dim > 0:
         components.append('input-node name=ivector dim=' + str(ivector_dim))
-        list.append('ReplaceIndex(ivector, t, 0)')
+        if ivector_interval == 0:
+            list.append('ReplaceIndex(ivector, t, 0)')
+            warnings.warn('You are using a single ivector for the entire chunk.')
```

Comment:
<code> warnings.warn('A single ivector will be used for the entire chunk as ivector_interval is 0. We recommend using an ivector interval of 10.') </code>